### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by JustOff**
1) add missed deniedPortAccess.longDesc to netError.dtd @ en-US
2) update crownin-fx.sh
3) update ru locale, based of new crownin-fx.sh

### CODE REVIEW

File: ru/browser/chrome/AB-CD/locale/browser/netError.dtd [hunk 1]
```diff
@@ -128,7 +128,12 @@ functionality specific to firefox. -->
 
 <!ENTITY securityOverride.warningContent "
 <p>Вам не следует добавлять исключение, если вы используете соединение с Интернетом, которому не полностью доверяете или если вы обычно не видите предупреждения при соединении с этим сервером.</p>
-<p>Если вы все же хотите добавить исключение для данного сайта, вы можете сделать это в дополнительных настройках шифрования.</p>">
+<p>Если вы все же хотите добавить исключение для данного сайта, вы можете сделать это в дополнительных настройках шифрования.</p>
+
+<button id='getMeOutOfHereButton'>&securityOverride.getMeOutOfHereButton;</button>
```
Comment (wolfbeast) [inline ru/browser/chrome/AB-CD/locale/browser/netError.dtd]: This content doesn't belong in a DTD file.

**Comment by JustOff** [comment]
But this content [present in en-US](https://github.com/MoonchildProductions/pale-moon-localization/blob/CrowdIn/en-US/browser/chrome/en-US/locale/browser/netError.dtd#L203), looks like it was missed in ru.

File: ru/browser/chrome/AB-CD/locale/browser/netError.dtd (hunk 1 above)
Comment (JustOff) [inline ru/browser/chrome/AB-CD/locale/browser/netError.dtd]: But this content [present in en-US](https://github.com/MoonchildProductions/pale-moon-localization/blob/CrowdIn/en-US/browser/chrome/en-US/locale/browser/netError.dtd#L203), looks like it was missed in ru.

Comment (JustOff) [inline ru/browser/chrome/AB-CD/locale/browser/netError.dtd]: More details: we have two netError.dtd files:

1) browser\chrome\AB-CD\locale\browser\
2) chrome\AB-CD\locale\AB-CD\global\

In the first one we have buttons, but in the second - not. This is how is done in en-US tree, so I updated ru, to be in accordance.

**Comment by JustOff** [comment]
Could you please review this request once again or advise how to move on?

**Comment by wolfbeast** [comment]
Context:
````
<!-- LOCALIZATION NOTE (securityOverride.warningContent) - Do not translate the
contents of the <button> tags. It uses strings already defined above. The
button is included here (instead of netError.xhtml) because it exposes
functionality specific to firefox. -->
````
The buttons do belong there in /browser. It's a really weird way of doing things but it's because it's an override of toolkit behavior.

File: ru/browser/chrome/AB-CD/locale/browser/netError.dtd (hunk 1 above)
Comment (wolfbeast) [inline ru/browser/chrome/AB-CD/locale/browser/netError.dtd]: it's actually correct. the /toolkit one is the base one, the /browser one overrides toolkit display content and Mozilla decided to plant XUL buttons in the DTD definition that should NOT be touched.

**Pull request merged**